endpoints/openrtb2/amp_auction.go

```diff
@@ -575,3 +583,27 @@ func setEffectiveAmpPubID(req *openrtb2.BidRequest, account string) {
 		}
 	}
 }
+
+// Update config channel object with information found from request. If no info found, we set channel name to "amp"
+func getChannelObjectAmpFromRequest(request []byte) (openrtb_ext.ExtRequestPrebidChannel, error) {
+	channelInfo, dataType, _, err := jsonparser.Get(request, "ext", "prebid", "channel")
```

Comment:
Each call to `jsonparser.Get` has to reparse the entire request. It's best to avoid this approach whenever possible. I think it would be more appropriate to use the `RequestWrapper` here, since it already has support for accessing the prebid request extension.

Reply:
Request wrapper is now utilized in latest commit!